inc/osvr/Server/ConfigureServerFromFile.h

```diff
@@ -65,115 +48,103 @@ namespace server {
+            log->info() << "Loading plugins...";
             srvConfig.loadPlugins();
             if (!srvConfig.getSuccessfulPlugins().empty()) {
-                out << "Successfully loaded the following plugins:" << endl;
+                log->info() << "Successfully loaded the following plugins:";
                 for (auto const &plugin : srvConfig.getSuccessfulPlugins()) {
-                    out << " - " << plugin << endl;
+                    log->info() << " - " << plugin;
                 }
-                out << "\n";
             }
             if (!srvConfig.getFailedPlugins().empty()) {
-                out << "Failed to load the following plugins:" << endl;
+                log->warn() << "Failed to load the following plugins:";
                 for (auto const &pluginError : srvConfig.getFailedPlugins()) {
-                    out << " - " << pluginError.first << "\t"
-                        << pluginError.second << endl;
+                    log->warn() << " - " << pluginError.first << "\t"
+                        << pluginError.second;
                 }
-                out << "\n";
             }
-
-            out << "\n";
         }
 
         {
-            out << "Instantiating configured drivers..." << endl;
+            log->info() << "Instantiating configured drivers...";
             bool success = srvConfig.instantiateDrivers();
             if (!srvConfig.getSuccessfulInstantiations().empty()) {
-                out << "Successes:" << endl;
+                log->info() << "Successes:";
                 for (auto const &driver :
                      srvConfig.getSuccessfulInstantiations()) {
-                    out << " - " << driver << endl;
+                    log->info() << " - " << driver;
                 }
-                out << "\n";
             }
             if (!srvConfig.getFailedInstantiations().empty()) {
-                out << "Errors:" << endl;
+                log->error() << "Errors:";
                 for (auto const &error : srvConfig.getFailedInstantiations()) {
-                    out << " - " << error.first << "\t" << error.second << endl;
+                    log->error() << " - " << error.first << "\t" << error.second;
                 }
-                out << "\n";
             }
-            out << "\n";
         }
 
         if (srvConfig.processExternalDevices()) {
-            out << "External devices found and parsed from config file."
-                << endl;
+            log->info() << "External devices found and parsed from config file.";
         }
 
         if (srvConfig.processRoutes()) {
-            out << "Routes found and parsed from config file." << endl;
+            log->info() << "Routes found and parsed from config file.";
         }
 
         if (srvConfig.processAliases()) {
-            out << "Aliases found and parsed from config file." << endl;
+            log->info() << "Aliases found and parsed from config file.";
```

Comment:
some of these messages are pretty dull, like this one, and could probably be reduced in severity, but we can do that after merge.